### DISCUSSION TITLE
9F3174FA-2355-B920-4B01-B5EFE5013267

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#101] [General]
# ACRIMSAT

ACRIMSAT is a NASA Earth observation satellite intended as a follow-up mission to the ACRIM instruments onboard SolarMax and UARS. It was launched on a Pegasus-XL rocket in 1999, and it continued to operate until 2013 when its batteries could no longer charge itself. The derelict spacecraft is still in orbit around Earth as of 2022, and will remain so for another 64 years after it lost contact

https://celestia.mobi/addon?item=9F3174FA-2355-B920-4B01-B5EFE5013267&lang=zh_CN

**Comment by Nicholas1401** [comment]
WoW
